### PR TITLE
Add GitLab

### PR DESCRIPTION
**Pull request by winniehell**

### CODE REVIEW

**Comment by winniehell** [comment]
@MrSwitch I can't figure out how to run the demos locally. I have set up https://github.com/MrSwitch/node-oauth-shim now but still no data is loaded. :disappointed:

File: src/modules/gitlab.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
+(function(hello) {
+
+	// TODO: make host configurable
+	var host = 'https://gitlab.com';
+
+	hello.init({
+
+		gitlab: {
+
+			name: 'GitHub',
```
Comment (shatgupt) [inline src/modules/gitlab.js]: Gitlab?

Comment (winniehell) [inline src/modules/gitlab.js]: yep, thank you :wink:

**Comment by winniehell** [comment]
I don't think I'm going to finish this soon, so closing for now.

**Pull request closed without merging**